index.html

```diff
@@ -138,8 +141,9 @@
     <section id='abstract'>
       <p>
 This specification describes mechanisms for ensuring the authenticity and
-integrity of structured digital documents using cryptography, such as digital
-signatures and other digital mathematical proofs.
+integrity of Verifiable Credentials and other types of structured digital
+documents using cryptography, such as digital signatures and other digital
+mathematical proofs.
```

Comment:
```suggestion
integrity of Verifiable Credentials using cryptography and especially through the use of digital signatures and related mathematical proofs.
```

Reply:
-1, the removal of "and other types of structured digital documents using cryptography" is exactly the sort of goal post shifting that is of concern with this PR. The following modification to your proposal could work:

```suggestion
integrity of Verifiable Credentials and similar types of constrained digital
documents using cryptography, especially through the use of digital 
signatures and related mathematical proofs.
```

Reply:
Good with this